app/views/shared/_google_recaptcha.html.erb

```diff
@@ -0,0 +1,11 @@
+<script src="https://www.google.com/recaptcha/api.js"></script>
+<script>
+  function onSubmit(token) {
```

Comment:
@littleforest wondering if we should wrap this inside `content_for :javascript` to not render the JS in the middle of the HTML

Reply:
Tried this, but broke the reCAPTCHA so just going to merge as-is. Thanks for the suggestion.